MovingBlocks/TerasologyLauncher
# Update Ukrainian locale with current strings

### Contains

Brings the Ukrainian localization up to date (by adding 2 new strings 😅)

### How to test

Launch the launcher, switch the language to Ukrainian, then try and trigger an "insufficient space in the download folder" error to see if things look alright.

### Outstanding before merging

Nothing! But quick tip: fancy Unicode strings can be converted to a `\u1234`-like format via `"your_fancy_string_here".encode('unicode-escape')` in Python (or a similar function in Java) now that Transifex will not do it automatically.
